Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -352,6 +350,112 @@ int CheckDeviceType_Read(HANDLE &dev_handle, u8* buf, int attempts)
 	return read;
 }
 
+static void CheckDeviceTypeByExtension(HANDLE &dev_handle, bool &real_wiimote, bool &is_bb)
+{
+	// max_cycles insures we are never stuck here due to bad coding...
+	int max_cycles = 20;
+	u8 buf[MAX_PAYLOAD] = { 0 };
+
+	u8 const req_status_report[] = { WM_SET_REPORT | WM_BT_OUTPUT, WM_REQUEST_STATUS, 0 };
+	// The new way to initialize the extension is by writing 0x55 to 0x(4)A400F0, then writing 0x00 to 0x(4)A400FB
+	// 52 16 04 A4 00 F0 01 55
+	// 52 16 04 A4 00 FB 01 00
+	u8 const disable_enc_pt1_report[MAX_PAYLOAD] = { WM_SET_REPORT | WM_BT_OUTPUT, WM_WRITE_DATA, 0x04, 0xa4, 0x00, 0xf0, 0x01, 0x55 };
+	u8 const disable_enc_pt2_report[MAX_PAYLOAD] = { WM_SET_REPORT | WM_BT_OUTPUT, WM_WRITE_DATA, 0x04, 0xa4, 0x00, 0xfb, 0x01, 0x00 };
+
+	CheckDeviceType_Write(dev_handle,
+		disable_enc_pt1_report,
+		sizeof(disable_enc_pt1_report),
+		1);
+	CheckDeviceType_Write(dev_handle,
+		disable_enc_pt2_report,
+		sizeof(disable_enc_pt2_report),
+		1);
+
+	int rc = CheckDeviceType_Write(dev_handle,
+		req_status_report,
+		sizeof(req_status_report),
+		1);
+
+	while (rc > 0 && --max_cycles > 0)
+	{
+		if ((rc = CheckDeviceType_Read(dev_handle, buf, 1)) <= 0)
+		{
+			// DEBUG_LOG(WIIMOTE, "CheckDeviceType: Read failed...");
```

Comment:
The whole `CheckDeviceTypeByExtension` was refactored/copy&pasted from `CheckDeviceType` so thats not actually my code/change